src/main/java/com/endava/exercises/extra/DateTime/AgeTask.java

```diff
@@ -0,0 +1,19 @@
+package com.endava.exercises.extra.DateTime;
+
+import java.time.LocalDate;
+import java.time.Period;
+//Write a Java program to calculate your age.
+public class AgeTask {
+    public static void main(String[] args) {
+
+        LocalDate yourBday = LocalDate.of(1993, 10, 30);
+        String yourName = "Laura";
+
+        LocalDate now = LocalDate.now();
+
+        Period diff = Period.between(yourBday, now);
+
+        System.out.printf(yourName+" is  %d years, %d months and %d days old.",
```

Comment:
please apply a String.format to printed message

Reply:
done